common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -1,15 +1,15 @@
 ---
 metadata:
-    display_name: Custom Javascript Display and Grading
+    display_name: Custom JavaScript Display and Grading
     markdown: !!null
     showanswer: never
 data: |
       <problem>
           <p>
               In these problems (also called custom JavaScript problems or JS Input
               problems), you add a problem or tool that uses JavaScript in Studio.
-              Studio embeds the problem in an IFrame so that your students can
-              interact with it in the LMS. You can grade your students' work using
+              Studio embeds the problem in an IFrame so that your learners can
```

Comment:
@catong I think I am going to leave it as IFrame as IFrame is used several other places in this file in the documentation. And honestly, as a developer I know immediately what an IFrame is, but an "inline frame" causes me to scratch my head.

IMHO it's OK for this documentation to be different from what is the partner documentation.